beacon-chain/rpc/prysm/v1alpha1/validator/proposer_epbs.go

```diff
@@ -31,6 +32,17 @@ func (vs *Server) SubmitSignedExecutionPayloadEnvelope(ctx context.Context, env
 	return nil, nil
 }
 
+// SubmitSignedExecutionPayloadHeader submits a signed execution payload header to the beacon node.
+func (vs *Server) SubmitSignedExecutionPayloadHeader(ctx context.Context, h *enginev1.SignedExecutionPayloadHeader) (*emptypb.Empty, error) {
```

Comment:
If I understood correctly, this is a method within beacon node and is intended to be called by validator, right?